mock/editors/reviewing.md

```diff
@@ -0,0 +1,28 @@
+# Reviewing Documents
+## Menu
+* [Group Reviews](#) (below)
```

Comment:
Suggest adding a section on "Explainers" as a useful precursor to all forms of review, helping people orient themselves to the problem being solved and the proposed solution approach; useful in general, required for TAG review anyway. Good practice to do it sooner rather than later.

Reply:
Added! Great idea!